sky/clouds/lambda_cloud.py

```diff
@@ -16,18 +16,36 @@
     'lambda_keys',
 ]
 
-# Currently, none of clouds.CloudImplementationFeatures are implemented
-# for Lambda Cloud
-_LAMBDA_IMPLEMENTATION_FEATURES: Set[clouds.CloudImplementationFeatures] = set()
-
 
 @clouds.CLOUD_REGISTRY.register
 class Lambda(clouds.Cloud):
     """Lambda Labs GPU Cloud."""
 
     _REPR = 'Lambda'
+    # Lamdba has a 64 char limit for cluster name.
+    # Reference: https://cloud.lambdalabs.com/api/v1/docs#operation/launchInstance # pylint: disable=line-too-long
+    _MAX_CLUSTER_NAME_LEN_LIMIT = 64
```

Comment:
For all such Cloud's own limits, don't the `ray-` prefix and suffix occupy some chars? Can we test close to the limit for all 4 clouds?